File: hysKim1/5622.py
#5622   다이얼
s = ['abc','def','ghi','jkl','mno','pqrs','tuv','wxyz']
input = input().lower()
min_time = 0
for i in range(len(input)):
    for j in s:
        if input[i] in j:
            min_time += s.index(j)+3
print(min_time)

File: yu-jin-song/step7/_5622.java
import java.util.Scanner;

public class _5622 {
	public static void main(String[] args) {
		String[] dial = {"", "", "ABC", "DEF", "GHI", "JKL", "MNO", "PQRS", "TUV", "WXYZ"};
		
		Scanner sc = new Scanner(System.in);
		
		String num = sc.nextLine();
		int time = 0;
		
		for(int i=0; i<num.length(); i++) {
			for(int j=2; j<dial.length; j++) {
				if(dial[j].contains(Character.toString(num.charAt(i)))) {
					time += 1+j;
					break;
				}
			}
		}
		
		System.out.println(time);
	}
}
